lib/util/package-rules/base-branches.ts

```diff
@@ -0,0 +1,29 @@
+import is from '@sindresorhus/is';
+import type { PackageRule, PackageRuleInputConfig } from '../../config/types';
+import { configRegexPredicate } from '../regex';
+import { Matcher } from './base';
+
+export class BaseBranchesMatcher extends Matcher {
+  static readonly id: string = 'base-branches';
```

Comment:
i don't see a value for those id's, ad they are only used in the api map, but then we only use the `map.values`